neuro_san/http_sidecar/handlers/connectivity_handler.py

```diff
@@ -41,8 +42,7 @@ async def get(self, agent_name: str):
         self.logger.info(metadata, "Start GET %s/connectivity", agent_name)
         try:
             data: Dict[str, Any] = {}
-            grpc_session: AsyncAgentSession = self.get_agent_grpc_session(metadata, agent_name)
-            result_dict: Dict[str, Any] = await grpc_session.connectivity(data)
+            result_dict: Dict[str, Any] = await service.connectivity(data, metadata)
 
```

Comment:
Instead of delegating request to gRPC session to local gRPC endpoint, use instance of AsyncAgentService, which we now have available locally.